web/wp-content/themes/dctx-v2/inc/acf-gutenberg.php

```diff
@@ -0,0 +1,443 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-wkeije.php, but found phpcs-scan-wKeIje.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).